cmake/sysbuild/suit.cmake

```diff
@@ -0,0 +1,291 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+include(${CMAKE_CURRENT_LIST_DIR}/suit_utilities.cmake)
+
+set(SUIT_GENERATOR_BUILD_SCRIPT "${ZEPHYR_SUIT_GENERATOR_MODULE_DIR}/ncs/build.py")
+set(SUIT_GENERATOR_CLI_SCRIPT "${ZEPHYR_SUIT_GENERATOR_MODULE_DIR}/suit_generator/cli.py")
+set(SUIT_OUTPUT_ARTIFACTS_DIRECTORY "DFU")
```

Comment:
This lines can and should be moved into suit_utilities.cmake, as that file uses them.

Reply:
Fixed!